decidim-conferences/db/migrate/20221118203655_add_weight_field_to_conferences.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddWeightFieldToConferences < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_conferences, :weight, :integer, null: false, default: 0
```

Comment:
If fix PRs we should not add any new migrations. It makes it difficult to backport.

Also, I don't think we actually need to add this column.